### PR TITLE
Welcome the GSoC 2017 students

### PR DESCRIPTION
**Pull request by gsamokovarov**
A blog post to officially welcome the 2017 students.

### CODE REVIEW

File: _posts/2017-05-08-gsoc-2017-students.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Rails. He will be mentored by [Jon], [Robin] and [Noah].
+
+## [Assain Jaleel][Assain]
+
+[Assain] is going to refresh the cookie system in Rails. He'll be mentored by
+[Kasper] with help from [Michael Coyne].
+
+## Thank You
+
+We'd like to shout-out to everyone who participated in this year selection
```
Comment (alexcameron89) [inline _posts/2017-05-08-gsoc-2017-students.markdown]: year -> year's

**Review by alexcameron89 — CHANGES_REQUESTED** [review]
I only found one change, otherwise it looks good.

File: _posts/2017-05-08-gsoc-2017-students.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Google Summer of Code 2017 Students
+category: activism
+author: Genadi Samokovarov
+published: true
+date: 2017-05-08 18:00:00 -02:00
+---
+
+We super are excited to announce our accepted Google Summer of Code students
```
Comment (robin850) [inline _posts/2017-05-08-gsoc-2017-students.markdown]: I guess it's the other way around: "We are super excited to ...". Nice post so far ! 👍

File: _posts/2017-05-08-gsoc-2017-students.markdown (hunk 1 above)
Comment (gsamokovarov) [reply _posts/2017-05-08-gsoc-2017-students.markdown]: Thanks!

File: _posts/2017-05-08-gsoc-2017-students.markdown (hunk 2 above)
Comment (gsamokovarov) [reply _posts/2017-05-08-gsoc-2017-students.markdown]: Yes it is, thanks for catching!

**Review by rafaelfranca — APPROVED** [review]

File: _posts/2017-05-08-gsoc-2017-students.markdown [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Google Summer of Code 2017 Students
+category: activism
+author: Genadi Samokovarov
+published: true
+date: 2017-05-08 18:00:00 -02:00
```
Comment (rafaelfranca) [inline _posts/2017-05-08-gsoc-2017-students.markdown]: Can you update the date so I can merge and publish?

Comment (gsamokovarov) [reply _posts/2017-05-08-gsoc-2017-students.markdown]: Sure, updated it. 😊

**Pull request merged**